### PR TITLE
Feature/codeclimate2

### PR DESCRIPTION
**Pull request by mhuseinov**
updated code climate to use rule of third for duplication and similarities

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -24,12 +24,17 @@ checks:
...
-      threshold: 100
+# https://docs.codeclimate.com/docs/duplication
+plugins:
+  duplication:
+    enabled: true
+    config:
+      count_threshold: 3
+      languages:
+        typescript:
+          mass_threshold: 80
```
Comment (pdittaro) [inline .codeclimate.yml:35]: Do we want to adjust this threshold up? Are all the duplications identified valid?

Comment (mhuseinov) [reply .codeclimate.yml:35]: At first glance it looks like they are valid enough. However, we may want to add specific rules to exclude some language constructions, for example, class/function initialization, filters etc. The overall amount of smells dropped from 190 to 129

Comment (pdittaro) [reply .codeclimate.yml:35]: Would be nice if we can drop it even more in a subsequent pr

**Review by pdittaro — APPROVED** [review]

**Review by dhaselhan — APPROVED** [review]
LGTM 👍

**Pull request merged**